src/ErrorBoundary/index.tsx

```diff
@@ -5,7 +5,7 @@ import FallbackComponent, {
 } from './FallbackComponent'
 
 export type Props = {
-  children: JSX.Element,
+  children: Exclude<React.ReactNode, number | string | boolean | null | undefined>,
```

Comment:
I think we can make this a bit cleaner with `NonNullable`:

```suggestion
  children: Exclude<NonNullable<ReactNode>, string | number | boolean>,,
```